front/app/containers/Admin/projects/all/index.tsx

```diff
@@ -92,7 +92,8 @@ const AdminProjectsList = memo(({ className }: Props) => {
   const userIsAdmin = isAdmin(authUser);
 
   // Fetch the admin publications to show in the 'Your projects' tab,
-  // including the (unexpanded) folders the user is a moderator of.
+  // including the (unexpanded) folders the user is a moderator of,
+  // but excluding the projects in those folders.
```

Comment:
This is confusing. The projects are there, just inside the folder, right? How about "excluding the projects that are already in the folders, avoiding duplicates"

Reply:
No, the projects in moderated folders are actually not in the data that is fetched in this constant.

Reply:
Okay, confusing but I guess a dev can figure it out with some extra work.